### ISSUE TITLE
weiried about the test PSNR

### ISSUE THREAD

**Issue by haifangqin** [JihyongOh/JSI-GAN#21]
hi, thanks for your great works!

I am now trying to test the pre-trained model you provide.

and when I test the testset you provide I got the PSNR about 35.77

while I test the LG_Daylight video 1000th-1050th frames , the PSNR is around 26-28db.

I download the 4k media from 4kmedia.org and extract frames as you say and save as .mat file to get the quantitative results.

Could you help me with that.  thanks a lot!  @sooyekim

**Comment by sooyekim** [comment]
26-28 dB seems a bit low. It would be worth checking if the input is indeed an SDR video frame and the GT is indeed an HDR frame.

**Comment by haifangqin** [comment]
thanks for your reply! I am also curious about it. but I use the same way to visualize the generated HDR mat file and the HDR mat you provided. the pic looks like an HDR image

Could you provide a whole test image mat file in the video named LG_Daylight_demo? We could check use this file.

**Comment by sooyekim** [comment]
LG_Daylight is already included in our test set. You could compare with frames 1-4 in the .mat or .png files.
